dotcom-rendering/src/web/components/Carousel.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
 import { breakpoints } from '@guardian/source-foundations';
-import { Carousel } from './Carousel';
+import { Carousel } from './Carousel.importable';
```

Comment:
Previously (and elsewhere) we hydrate this component as part of `OnwardsUpper`. But here it sits alone and needs hydrating by itself